### PR TITLE
[plugin.video.mazacka]v2.0.0

### PR DESCRIPTION
**Pull request by homolaradek**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by enen92** [comment]
@homolaradek check Travis error. The same version already exists in Leia, matrix has to be higher for users to migrate

File: plugin.video.mazacka/addon.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <description lang="cs_CZ">Mazací tramvaj s evidenčním číslem 5572, pro kterou se vžil zkrácený název "Mazačka", slouží na pražských tramvajových tratích od ledna 2015. Aplikace maziva pomáhá snížit hluk a opotřebení kolejnic a kol. Každý den najede Mazačka v průměru přes 200 kilometrů.</description>
+  <description lang="sk_SK">Mazacie električka s evidenčným číslom 5572, pre ktorú sa vžil skrátený názov "Mazačka", slúži na pražských električkových tratiach od januára 2015. Aplikácia mazivá pomáha znížiť hluk a opotrebenie koľajníc a kolies. Každý deň urazí Mazačka v priemere vyše 200 kilometrov.</description>
+  <description lang="de_DE">Die Straßenbahn mit der Kennzeichennummer 5572, die die Schienen schmiert, für die der abgekürzte Name "Mazačka" übernommen wurde, wird seit Januar 2015 in Prager Straßenbahnlinien eingesetzt. Die Schmiermittelanwendung trägt dazu bei, Geräusche und Verschleiß an Schienen und Rädern zu reduzieren. Mazačka legt jeden Tag durchschnittlich über 200 Kilometer zurück.</description>
```
Comment (enen92) [inline plugin.video.mazacka/addon.xml:20]: License element is missing, please add one with the corresponding spdx identifier

File: plugin.video.mazacka/addon.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <platform>all</platform>
```
Comment (enen92) [inline plugin.video.mazacka/addon.xml:21]: Source is also missing, please add one that points to the upstream location of the code

**Review by enen92 — CHANGES_REQUESTED** [review]
Code is good to go, only info missing in addon.xml.

File: plugin.video.mazacka/addon.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <language>cs en de sk</language>
```
Comment (enen92) [inline plugin.video.mazacka/addon.xml]: This is the language of the content not the translations. Does the actual media have all these languages? If not remove the ones that do not apply

Comment (homolaradek) [reply plugin.video.mazacka/addon.xml]: Okay, as it's video live stream without any speech content or subtitles, I removed this tag.

**Review by enen92 — APPROVED** [review]

**Pull request merged**